.travis.yml

```diff
@@ -2,10 +2,10 @@
 language: julia
 os:
   - linux
-  - osx
+#  - osx
 julia:
   - 0.6
-  - nightly
+  - 0.7
```

Comment:
it's best practice to leave this as nightly.

Reply:
I don't actually think this will work as 0.7 until there's a release.

Reply:
9b5191f571a1930329899921e4fb86577fb5020c